R/features.R

```diff
@@ -135,11 +133,12 @@ feature_not_exact <- function(m, n_combinations = 200, weight_zero_m = 10^6, red
   X[, ind := NULL]
 
   # Set column order and key table
-  nms <- c("ID", "features", "nfeatures", "N", "shapley_weight", "no")
+  nms <- c("ID", "features", "nfeatures", "N", "shapley_weight")
   data.table::setcolorder(X, nms)
   data.table::setkey(X, nfeatures)
   X[, ID := .I]
   X[, N := as.integer(N)]
+  X[, no := 1] # TODO: Remove this after updating test objects
```

Comment:
I don't understand why you have put this here. 
I think the crucial thing to check is that with the previous code and reduce_dim = TRUE, no will be equal to rep(1,nrow(X)) both when there are indeed duplicates and when there are no duplicates. I don't know if you have checked that, but if not, I think it is worth checking that with an example.

Reply:
Oh, this was just to avoid that the tests failed.